src/plugins/order_stash.coffee

```diff
@@ -0,0 +1,346 @@
+  }
+
+  #sa-order-stash-rationale {
+    display: none;
+    padding: 0 15px;
+    text-align: left;
+  }
+
+  #sa-order-stash-rationale p:first-child {
+    font-size: 14px;
+  }
+
+  #sa-order-stash-rationale p {
+    margin-bottom: 23px;
+    line-height: 17px;
+    font-family: Verdana, Arial, sans-serif;
+    font-size: 13px;
+  }
+
+  #sa-order-stash-rationale p.sa-order-stash-privacy {
+    font-size: 11px;
+    color: #909090;
+    margin-bottom: 0;
+  }
+
+  @media only screen and (-webkit-min-device-pixel-ratio: 2),
+         only screen and (min--moz-device-pixel-ratio: 2),
+         only screen and (  -o-min-device-pixel-ratio: 2/1),
+         only screen and (     min-device-pixel-ratio: 2),
+         only screen and (     min-resolution: 192dpi),
+         only screen and (     min-resolution: 2dppx) {
+
+    #sa-order-stash-header {
+      background-image: url('#{asset_url("logo@2x.png")}');
+    }
+
+    #sa-order-stash-dismiss {
+      background-image: url('#{asset_url("btn_close@2x.png")}');
+    }
+
+    #sa-order-stash-button {
+      background-image: url('#{asset_url("arrow@2x.png")}');
+    }
+  }
+  """
+
+  TEMPLATE = (assigns) -> """
+  <div id="sa-order-stash-header">
+    <span id="sa-order-stash-dismiss"></span>
+  </div>
+
+  <div id="sa-order-stash-content">
+    <p id="sa-order-stash-prompt">
+      Αποθήκευση της παραγγελίας σου στο skroutz;
+    </p>
+
+    <div id="sa-order-stash-rationale">
+      <p>
+        Γιατί να αποθηκεύσω την παραγγελία μου στο skroutz;
+      </p>
+
+      <p>
+        Ώστε να μπορείς να ανατρέξεις πληροφορίες σχετικά με τις αγορές που έκανες,
+        μέσα στο λογαριασμό σου στο skroutz.
+      </p>
+
+      <p class="sa-order-stash-privacy">
+        Τα δεδομένα αυτά είναι προσωπικά και παραμένουν ασφαλή στο skroutz.
+      </p>
+    </div>
+  </div>
+  <a id="sa-order-stash-button" class="read-less" href="#{assigns.endpoint}">Αποθήκευση</a>
+  <a id="sa-order-stash-why" href="javascript:void(0)">Γιατί να την αποθηκεύσω;</a>
+  """
+
+  constructor: ->
+    @addStyle()
+    @render()
+    @_bindHandlers()
+
+  ###
+  Displays the plugin
+  ###
+  render: ->
+    @$el = document.createElement('div')
+    @$el.id = 'sa-order-stash-plugin'
+    @$el.innerHTML = TEMPLATE(endpoint: @_endpoint())
+    @_document().body.appendChild(@$el)
+
+  $header: -> @_document().getElementById('sa-order-stash-header')
+  $dismissButton: -> @_document().getElementById('sa-order-stash-dismiss')
+  $stashButton: -> @_document().getElementById('sa-order-stash-button')
+  $whyButton: -> @_document().getElementById('sa-order-stash-why')
+  $prompt: -> @_document().getElementById('sa-order-stash-prompt')
+  $rationale: -> @_document().getElementById('sa-order-stash-rationale')
+
+  ###
+  Adds the required css for the plugin
+  ###
+  addStyle: ->
+    style = window.document.createElement('style')
+    style.id = 'sa-order-stash-style'
+    style.type = 'text/css'
+
+    if style.styleSheet
```

Comment:
I think this condition is for browser compatibility, i suggest to add a comment here.